### PR TITLE
Fixes failing legs code

### PR DESCRIPTION
**Pull request by QualityVan**
## About The Pull Request
Any two legs/feet that were missing or broken were supposed to cause an occasional knockout but this usually didn't happen because missing limbs don't process. Fixes that, and also reduces the knockout from a 20 second unconscious to a 3 second knockdown. It was a little overkill.
Also fixes amputated limbs not being amputated.

## Why It's Good For The Game
Bugfixes, and a random 20 second unconscious is not necessary.

## Changelog
:cl:
fix: When you have two broken or missing legs or feet, you'll now fall down once or twice a minute.
fix: Amputated limbs are amputated.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/life/handle_organs.dm [hunk 1]
```diff
@@ -31,6 +31,11 @@
 
 	for(var/i in limbs)
 		var/datum/limb/E = i
+
+		if((E.name in list("l_leg", "l_foot", "r_leg", "r_foot")) && !lying_angle) //Need to do this before checking need_process in order to catch missing limbs
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/human/life/handle_organs.dm:35]: O know this is just moved code but my eyes

File: code/modules/organs/limbs.dm [hunk 2]
```diff
@@ -666,7 +666,10 @@ Note that amputating the affected organ does in fact remove the infection from t
 	if(body_part == CHEST)
 		return FALSE
 
-	set_limb_flags(LIMB_DESTROYED)
+	if(amputation)
+		set_limb_flags(LIMB_AMPUTATED & LIMB_DESTROYED)
```
Comment (TiviPlus) [inline code/modules/organs/limbs.dm:670]: this is the same as
set_limb_flags(0) do you mean |

**Review by TiviPlus — APPROVED** [review]

**Review by TiviPlus — CHANGES_REQUESTED** [review]
.

File: code/modules/organs/limbs.dm (hunk 2 above)
Comment (QualityVan) [reply code/modules/organs/limbs.dm:670]: i sure do whoops

**Pull request merged**